testdata/repositories/bitkeeper/BitKeeper/etc/config

```diff
@@ -0,0 +1,182 @@
+# This is the BitKeeper configuration for this package.
+# 
```

Comment:
As mentioned [by vladak](https://github.com/OpenGrok/OpenGrok/pull/1456#issuecomment-288805553) we do not deliver dot dirs in the repositories.

Check these sections:
 - for ant builds:
   - [repository packing](https://github.com/OpenGrok/OpenGrok/blob/master/build.xml#L770-L798)
   - [repository cleanup](https://github.com/OpenGrok/OpenGrok/blob/master/build.xml#L885-L892)
 - for maven (travis) builds
   - [repository packing](https://github.com/OpenGrok/OpenGrok/blob/master/opengrok-indexer/build.xml#L101-L133)
   - [repository cleanup](https://github.com/OpenGrok/OpenGrok/blob/master/opengrok-indexer/build.xml#L185-L192)